packages/algob/src/lib/tx.ts

```diff
@@ -247,3 +247,119 @@ export async function transferASALsig (
 
   return await deployer.waitForConfirmation(tx1.txId);
 }
+
+/**
+ * Description: Make calls to application
+ * @param deployer AlgobDeployer
+ * @param sender Account from which call needs to be made
+ * @param payFlags Transaction Flags
+ * @param appId ID of the application being configured or empty if creating
+ */
+export async function callNoOpSSC (
```

Comment:
Why this is called `NoOp`? Is this a no operational call? will this call do any changes? Please update the description accordingly

Reply:
`NoOp - Generic application calls to execute the ApprovalProgram` this is as per https://developer.algorand.org/docs/features/asc1/stateful/#the-lifecycle-of-a-stateful-smart-contract, updating description